packages/invoice-dashboard/package.json

```
@@ -1,6 +1,6 @@
 {
   "name": "@requestnetwork/invoice-dashboard",
-  "version": "0.3.1",
+  "version": "0.3.2",
```

Please make this a minor version increase as well instead of a patch
```suggestion
  "version": "0.4.0",
```